specification/compute/resource-manager/Microsoft.Compute/stable/2021-08-01/disk.json

```diff
@@ -2476,6 +2493,10 @@
           {
             "value": "EncryptionAtRestWithPlatformAndCustomerKeys",
             "description": "Disk is encrypted at rest with 2 layers of encryption. One of the keys is Customer managed and the other key is Platform managed."
+          },
+          {
+            "value": "ConfidentialVmEncryptedWithCustomerKey",
+            "description": "Confidential VM supported disk encrypted with customer managed key."
```

Comment:
same as below.. needs to be more descriptive. Please ask feature PM for pointers

Reply:
checked with PM, and modified to -
"Confidential VM supported disk and VM guest state would be encrypted with customer managed key."

Also removed "ConfidentialVmEncryptedWithCustomerKey" from "EncryptionType" enum, as it is used at disk level. (not for DES type)